RNA-seq/02-gastric_cancer_tximport-live.Rmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "Gastric cancer: gene-level summarization with `tximport`"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2018**
+
+In this notebook, we'll import transcript level output from `salmon quant` and
+summarize it to the gene level using [`tximport`](https://bioconductor.org/packages/release/bioc/html/tximport.html).
+
+![](diagrams/rna-seq_5.png)
+
+For more information about `tximport`, see [this excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html) from Love, Soneson, and Robinson.
+
+We'll need the `quant.sf` files for all the samples in an experiment and a file
+that maps between Ensembl transcript ids and Ensembl gene ids.
+In refine.bio, we generate these along side the transcriptome indices for an
+organism.
+
+## Libraries and functions
+
+```{r}
+# Load the tximport package
+
+```
+
+## Directories and files
+
+```{r}
+# directory where the quant files are located, each sample is its own
+# directory
+quant_dir <- file.path("data", "quant", "gastric_cancer")
+
+# create a directory to hold the tximport results if it doesn't exist yet
+
+
+# the quant files themselves
+sf_files <- list.files(quant_dir, recursive = TRUE, full.names = TRUE,
+                       pattern = "quant.sf")
+
+# file that contains the gene to transcript mapping from refine.bio
+tx2gene_file <- file.path("index", "Homo_sapiens",
+                          "Homo_sapiens.GRCh38.95_tx2gene.tsv")
+```
+
+**Output**
+
+```{r}
+# Name the output gastric_cancer_tximport.RDS and get ready to save it in the 
```

Comment:
I know this is a note for you, but what specifically do you mean by "get ready to save it"? I assume you mean make the correct path for it?